src/hotspot/share/opto/cfgnode.cpp

```diff
@@ -394,6 +394,45 @@ bool RegionNode::is_unreachable_from_root(const PhaseGVN* phase) const {
   return true; // The Region node is unreachable - it is dead.
 }
 
+// Is this region in an infinite subgraph?
+// (no path to root except through false NeverBranch exit)
+bool RegionNode::is_in_infinite_subgraph() {
```

Comment:
It is used only in assert(). Should it be under `#ifdef ASSERT` ?

Reply:
:+1: